src/Common/src/System/NotImplemented.cs

```diff
@@ -21,11 +21,5 @@ internal static Exception ByDesignWithMessage(string message)
         {
             return new NotImplementedException(message);
         }
-
-        /// <summary>
-        /// Temporary NotImplementedException with no message shown to user.
-        /// Example: Exception.ActiveIssue("https://github.com/dotnet/corefx/issues/xxxx") or Exception.ActiveIssue("TFS xxxxxx").
-        /// </summary>
-        internal static Exception ActiveIssue(string issue) => new NotImplementedException();
```

Comment:
We don't want to delete this. Even if it's not being used in this assembly (or even in any assembly), it's a point-in-time condition - we keep that around to mark `NotImplementedExceptions` that are tracked by an active issue.